pages/index.js

```diff
@@ -1,23 +1,24 @@
 import Head from "next/head";
 import Header from "../components/Header";
 import Footer from "../components/Footer";
-import Cards from "../components/Cards";
+import NavBar from "../components/NavBar";
 
 // Importing stylesheets
 import globalStyles from "../styles/global";
 
 export default function Home() {
-    return (
-        <div className="container">
-            <Head>
-                <title>Evolution</title>
-                <link rel="icon" href="/favicon.ico" />
-            </Head>
-            <Header />
-            <Footer />
-            <style jsx global>
-                {globalStyles}
-            </style>
-        </div>
-    );
+  return (
+    <div className="container">
+      <Head>
+        <title>Evolution</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <NavBar />
+      <Header />
+      <Footer />
+      <style jsx global>
+        {globalStyles}
+      </style>
```

Comment:
I think you can use your new `Page` component here.

```suggestion
      <Page>
        <Header />
        <Footer />
      </Page>
```

Reply:
addressed on the last commit